python.d/neo.chart.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+# Description:  netdata python.d module for monitoring Neo4j
+# Author: Jerome Baton, @wadael , copied from the example of Pawel Krupa (paulfantom)
+
+from bases.FrameworkServices.SimpleService import SimpleService
+
+try:
+    from neo4j.v1 import GraphDatabase
+    NEO4JDRIVER = True
+except ImportError:
+    NEO4JDRIVER = False
+
+
+ORDER = ['monnodes','monlabels','monrels']
+
+CHARTS = {
+    'monnodes': {
+        'options': [None, 'Nodes count', 'Nodes', 'Node count', 'neo4j', 'line'],
+        'lines': [
+             ['id', 'name', 'absolute', 1, 1, 'hidden']
```

Comment:
Reason for `hidden` dimensions?

Reply:
Hi sorry for the delay.
The reason is I saw this in an example and I am pretty confident it did not work without.